web/cypress/support/const.ts

```diff
@@ -1,6 +1,6 @@
 //set all your shared const here
-export const url = 'http://localhost:9000/netflow-traffic';
-// export const url = 'http://localhost:9001';
+// export const url = 'http://localhost:9000/netflow-traffic';
+export const url = 'http://localhost:9001';
```

Comment:
is that an intentional change or something you forgot to switch back to original?

Reply:
I mostly use `standalone` for these so I guess it would be great to keep it as default